scripts/i2i/translate_image.py

```diff
@@ -16,50 +18,26 @@ def preprocess_mri(x):
     return x
 
 
-def main():
-    parser = argparse.ArgumentParser(description="Translate image.")
-    parser.add_argument(
-        "-m",
-        "--model",
-        dest="model",
-        type=str,
-        help="generator pth file",
-        default=r"E:\Develop\Scripts\ML-SEG\cyclegan\checkpoints\t1w2ctm\latest_net_G.pth",
-    )
-    parser.add_argument(
-        "-i", "--input", dest="input", type=str, required=True, help="input image (A)"
-    )
-    parser.add_argument(
-        "-o", "--output", dest="output", type=str, help="output image (B)"
-    )
-    parser.add_argument(
-        "--axis", type=int, help="translation applied on slice YZ=0, XZ=1, XY=2"
-    )
-    parser.add_argument(
-        "--debug_axis", action="store_true", help="extract center slice and exit"
-    )
-    parser.add_argument(
-        "--resample", action="store_true", help="resample input to target spacing"
-    )
-    parser.add_argument(
-        "--pad", action="store_true", help="pad to next multiple of 256 / 4"
-    )
-    parser.add_argument(
-        "--gpu_ids",
-        nargs="+",
-        type=int,
-        help="space seperated list of GPU ids",
-        default=[],
-    )
-    args = parser.parse_args()
+def main(
+    model: Path,
+    input: Path,
+    output: Path = Path("out.nii.gz"),
+    axis: int = 2,
+    debug_axis: bool = False,
+    gpu_ids: List[int] = [],
+):
+    """Translate image using style transfer model"""
+
+    # make sure input and output folder are valid
+    assert input.exists() and output.parent.exists()
 
-    if not args.input:
-        args.input = r"F:\Data\DRCMR-Thielscher\all_data\images\X10679.nii.gz"
+    if not gpu_ids:
```

Comment:
I guess you get that from the default, right? i.e. ``gpu_ids = [] `` You do not need these two likes

Reply:
done, thanks